.pre-commit-config.yaml

```diff
@@ -40,9 +36,8 @@ repos:
     hooks:
       - id: sort-all
         types: [file, python]
-  - repo: https://github.com/PyCQA/isort
-    rev: "5.12.0"
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: 'v0.0.292'
```

Comment:
v0.1.0 just dropped 👀
https://github.com/astral-sh/ruff/releases/tag/v0.1.0

Reply:
Bumped to v0.1.5 now.